lib/components/PlayerScreen/progress_slider.dart

```diff
@@ -37,6 +37,8 @@ class _ProgressSliderState extends State<ProgressSlider> {
 
     _sliderThemeData = SliderTheme.of(context).copyWith(
       trackHeight: 4.0,
+      inactiveTrackColor:
+          Theme.of(context).colorScheme.surfaceVariant.withAlpha(0x80),
```

Comment:
I've used `.withOpacity()` everywhere else so far, do you think that's fine or should I be using `.withAlpha()` instead?

Reply:
I didn't really think about it when using it 😅
So, it seems like they basically do the same, just that one takes an int from 0-255, the other a float. `withOpacity` actually calls `withAlpha` after multiplying by 255, so it has more overhead, but I think `withOpacity` is easier to read and understand, so I'll update my code to use that.

Reply:
Good to know. On the web in CSS it actually behaves differently, that's why I was curious. But yeah, a float should be easier then, thanks!

Reply:
Right, I read that when I was looking up   the Flutter method for it. The name difference is a tad confusing when it does the same but that's most likely because Dart doesn't support function overloading..